src/v1beta1/firestore_client.js

```diff
@@ -1,68 +1,25 @@
-/*
- * Copyright 2017, Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * EDITING INSTRUCTIONS
- * This file was generated from the file
- * https://github.com/googleapis/googleapis/blob/master/google/firestore/v1beta1/firestore.proto,
- * and updates to that file get reflected here through a refresh process.
- * For the short term, the refresh process will only be runnable by Google
- * engineers.
- *
- * The only allowed edits are to method and file documentation. A 3-way
- * merge preserves those additions if the generated source changes.
- */
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 'use strict';
 
-var configData = require('./firestore_client_config');
-var extend = require('extend');
-var gax = require('google-gax');
-
-var SERVICE_ADDRESS = 'firestore.googleapis.com';
-
-var DEFAULT_SERVICE_PORT = 443;
-
-var CODE_GEN_NAME_VERSION = 'gapic/0.0.5';
-
-var PAGE_DESCRIPTORS = {
-  listDocuments: new gax.PageDescriptor(
-    'pageToken',
-    'nextPageToken',
-    'documents'
-  ),
-  listCollectionIds: new gax.PageDescriptor(
-    'pageToken',
-    'nextPageToken',
-    'collectionIds'
-  ),
-};
-
-var STREAM_DESCRIPTORS = {
-  batchGetDocuments: new gax.StreamDescriptor(gax.StreamType.SERVER_STREAMING),
-  runQuery: new gax.StreamDescriptor(gax.StreamType.SERVER_STREAMING),
-  write: new gax.StreamDescriptor(gax.StreamType.BIDI_STREAMING),
-  listen: new gax.StreamDescriptor(gax.StreamType.BIDI_STREAMING),
-};
-
-/*!
- * The scopes needed to make gRPC calls to all of the methods defined in
- * this service.
- */
-var ALL_SCOPES = [
-  'https://www.googleapis.com/auth/cloud-platform',
-  'https://www.googleapis.com/auth/datastore',
-];
+const gapicConfig = require('./firestore_client_config');
+const gax = require('google-gax');
+const merge = require('lodash.merge');
```

Comment:
Which is super strange since it seems that it's actually used here...  https://github.com/googleapis/nodejs-firestore/blob/fc139d07281331d47a6a75b3c0fbdb9ffb471352/src/v1beta1/firestore_client.js#L106

What version of `eslint` is this using?  If it's 5.x, can you try backing off to 4.x and seeing if you get the same result?